test/unit/attachment_test.rb

```diff
@@ -76,8 +68,8 @@ def test_diskfilename
       with_settings(:attachment_max_size => 0) do
         @issue = WorkPackage.find(1)
         response = Attachment.attach_files(@issue, {
-                                             '1' => {'file' => mock_file, 'description' => 'test'},
-                                             '2' => {'file' => mock_file, 'description' => 'test'}
+                                             '1' => {'file' => create_uploaded_file, 'description' => 'test'},
```

Comment:
Line is too long. [110/99]<br>Space inside { missing.<br>Space inside } missing.